client/src/components/ElementEditor/ElementEditor.js

```diff
@@ -128,6 +133,7 @@ ElementEditor.propTypes = {
   }),
 };
 
+const filteredElementFormStates = {};
```

Comment:
I tried add this to [the "graphql2" PR](https://github.com/silverstripe/silverstripe-elemental/pull/1175) however it caused the issue where submitting a page with an invalid elemental field and then inline saving did not pop open the preview

Don't try and make this code work, it's not necessary